src/engine/channelhandle.h

```diff
@@ -160,6 +159,10 @@ class ChannelHandleMap {
     typedef typename QVarLengthArray<T, kMaxExpectedGroups>::const_iterator const_iterator;
     typedef typename QVarLengthArray<T, kMaxExpectedGroups>::iterator iterator;
 
+    ChannelHandleMap()
+            : m_dummy(T()) {
```

Comment:
I suggest to replace `T()` with `T{}` (here and when appending) to improve readability and to indicate that this is a constructor initialization.